app/assets/stylesheets/_steps.scss

```diff
@@ -20,7 +21,14 @@ $dark-grey: #999999;
 }
 
 .markdown-example {
+  font-size: 14px;
```

Comment:
we could potentially use mixins from govuk-frontend for these: `@include govuk-font(19)`, `padding: govuk-spacing(4)`, etc.

Reply:
updated